components/nav/navItemWithSmallDropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from "react";
+import Link from "next/link";
+import {UpIcon} from "./UpIcon"
+import DownIcon from "./DownIcon"
+import Image from "next/image";
+
+export interface NavItemWithSmallDropdownProps {
```

Comment:
<img width="363" alt="image" src="https://github.com/keploy/website/assets/53110238/0428fb80-552f-4bec-bba5-ab292eb3ed25">

The dropdown overlay with the hero in terms of background, i think it's better to keep the dropdown background either with light grey or `#E6E2D4` color or when dropdown menu is clicked rest background get's a bit darker to put focus on resources section

Reply:
Hey @Sonichigo, I've updated the dropdown background to light grey to ensure it contrasts well with background. This should help in focusing on the resources section when the dropdown is active. Can you take a look?